pyccel/ast/core.py

```diff
@@ -4709,6 +4712,22 @@ def __getnewargs__(self):
         args = (self.accel, self.txt)
         return args
 
+class OMP_ForLoop(AnnotatedComment):
+    """ Represents and OpenMP Loop construct. """
```

Comment:
Typo `and OpenMP Loop` -> `an OpenMP Loop`

Reply:
Care to fix the typos?